dissect/target/plugins/os/unix/log/auth.py

```diff
@@ -45,7 +45,13 @@ def authlog(self) -> Iterator[AuthLogRecord]:
         var_log = self.target.fs.path("/var/log")
         for auth_file in chain(var_log.glob("auth.log*"), var_log.glob("secure*")):
             for ts, line in year_rollover_helper(auth_file, RE_TS, "%b %d %H:%M:%S", tzinfo):
-                message = line.replace(re.search(RE_TS_AND_HOSTNAME, line).group(0), "").strip()
+                ts_and_hostname = re.search(RE_TS_AND_HOSTNAME, line)
+                if not ts_and_hostname:
+                    self.target.log.warning(f"No timstamp and hostname found on one of the lines in %s.", auth_file)
```

Comment:
```suggestion
                    self.target.log.warning("No timstamp and hostname found on one of the lines in %s.", auth_file)
```

The f is no longer needed ;-)